metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/MathFunctions.java

```diff
@@ -25,17 +25,39 @@
 import org.apache.metron.stellar.dsl.StellarFunction;
 
 import java.util.List;
+import java.util.function.Function;
 
 public class MathFunctions {
 
-  @Stellar(name="ABS"
-          ,description="Returns the absolute value of a number."
-          ,params = {
-                "number - The number to take the absolute value of"
-                    }
-          , returns="The absolute value of the number passed in."
-          )
-  public static class Abs implements StellarFunction {
+  private enum SingleArgMathFunctions implements Function<Double, Double> {
+    ABS(d -> Math.abs(d)),
+    CEIL(d -> Math.ceil(d)),
+    COS(d -> Math.cos(d)),
+    FLOOR(d -> Math.floor(d)),
+    LOG10(d -> Math.log10(d)),
+    LOG2(d -> Math.log(d)/Math.log(2)),
+    LN(d -> Math.log(d)),
+    SIN(d -> Math.sin(d)),
+    SQRT(d -> Math.sqrt(d)),
+    TAN(d -> Math.tan(d)),
+    ;
+
```

Comment:
Is there a reason why only this subset of Math.* is exposed?  What about the rest?  Are they not needed, applicable, or left for future work?